### PR TITLE
LOOP-1287 Prefer instance identifiers to allow reuse of DeviceManager types

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-1287

DashPumpManager is being subclassed to provide a new demo pumpmanager that uses a mock sdk.

### CODE REVIEW

File: MockKit/MockCGMManager.swift [hunk 1]
```diff
@@ -254,7 +254,16 @@ extension MockCGMLifecycleProgress: RawRepresentable {
 public final class MockCGMManager: TestingCGMManager {
     
     public static let managerIdentifier = "MockCGMManager"
+
+    public var managerIdentifier: String {
+        return MockCGMManager.managerIdentifier
```
Comment (darinkrauss) [inline MockKit/MockCGMManager.swift:259]: [nit] If the static is no longer necessary, why not just collapse into:
```
public var managerIdentifier: String { "MockCGMManager" }
```
and remove the static?

Comment (darinkrauss) [reply MockKit/MockCGMManager.swift:259]: Here and below for `localizedTitle`. Also in `MockPumpManager.swift`.

**Review by darinkrauss — APPROVED** [review]
LGTM! 🤖

File: MockKit/MockCGMManager.swift (hunk 1 above)
Comment (ps2) [reply MockKit/MockCGMManager.swift:259]: @darinkrauss There is still code in the Mock devices that utilize the static, and that's ok, as they are not re-using types for multiple DeviceManager instances.

**Review by rickpasetto — APPROVED** [review]
LGTM 🥍

**Pull request merged**